filesystems/oss/src/main/scala/cromwell/filesystems/oss/nio/TTLOssStorageConfiguration.scala

```diff
@@ -0,0 +1,52 @@
+package cromwell.filesystems.oss.nio
+
+import com.aliyun.oss.OSSClient
+import com.typesafe.config.{Config, ConfigFactory}
+import net.ceedubs.ficus.Ficus._
+
+object TTLOssStorageConfiguration {
+  def config = {
+    ConfigFactory.invalidateCaches
```

Comment:
After discussions within the DSP-Batch team, we do not want `invalidateCaches()` to be running after Cromwell has completed startup, especially repeatedly.

Instead, how about logic something like:
- Replace (or in-addition-to?) the embedded config value `auth.security-token` there's an `auth.security-token-file` which represents a file path containing the token
- The Cromwell code rereads the `auth.security-token-file` contents every `refresh-interval`
    - Maybe the timer config is renamed to something like `auth.security-token-file-ttl`?
    - The interval defaults to `None` that translates to `Duration.Inf`?
    - If it turns out more that the token needs to be reloaded, say the `endpoint`/`accessId`/`accessKey` then those could also live in the same external file, and something uncached like `ConfigFactory.parseFile()` could be used.

Again, something like the above has the benefits of:
1. The *entire* cromwell config is not re-loaded
2. The `auth.security-token-file` can be still be swapped in by some external process
    - write: `echo "MyToken" > sts.tmp`
    - swap: `mv sts.tmp sts`

Naming is not my strong suit so I'm open to other ideas. It's also fine if this logic is all merged back into one `OssStorageConfiguration`, or leave it separate if you think it looks cleaner. Again, mainly we don't want `ConfigFactory.invalidateCaches()` called.

Reply:
Thanks for your suggestion !
This is just an embedded framework of credential refreshing. The `invalidateCaches ` logic could be removed currently, and a more graceful implement will be provided in next version, instead of reloading config file.

Reply:
> a more graceful implement will be provided in next version

Does the "next version" mean another push for this pull-request, or does "next version" mean we should review and merge this as-is and then a future pull-request will make changes?

Reply:
The latter, a future pull-request.